src/coreclr/tools/Common/TypeSystem/IL/HelperExtensions.cs

```diff
@@ -40,6 +40,13 @@ public static MethodDesc GetOptionalHelperEntryPoint(this TypeSystemContext cont
             return helperMethod;
         }
 
+        public static MethodDesc GetHelperEntryPoint(this TypeSystemContext context, string typeNamespace, string typeName, string methodName)
```

Comment:
Not a fan of this. This is more like "get arbitrary thing in corelib".

I see we already have to do `context.SystemModule.GetKnownType("System", "Math")` in the only place that uses this anyway. I'd just create a static local function to find a method on `System.Math` there and use that.

Reply:
Removed.